### ISSUE TITLE
Fab buttons will be displayed under the ion-item-divider when not sticky

### ISSUE THREAD

**Issue by alan-agius4** [driftyco/ionic#8489]
Fab buttons will be displayed under the ion-item-divider when not sticky, As ion-item-divider has a z-index even when not sticky

#### Short description of the problem:
z-index is applied on ion-item-divider when might not be needed when it's not sticky

#### What behavior are you expecting?
No z-index applied and Fab button to be displayed on top of the ion-item-divider

![screen shot 2016-10-04 at 23 59 30](https://cloud.githubusercontent.com/assets/17563226/19094323/d6b0943c-8a8f-11e6-8094-ea5f55a2a637.png)

Ionic 2.

**Comment by manucorporat** [comment]
@alan-agius4 what issue is causing to have z-index in non-sticky item dividers?

**Comment by alan-agius4** [comment]
fab buttons will show under it, sorry forgot the screenshot

![screen shot 2016-10-04 at 23 59 30](https://cloud.githubusercontent.com/assets/17563226/19094114/bad5c396-8a8e-11e6-80d4-279d5a59dcc3.png)

**Comment by manucorporat** [comment]
ok, looks like this is a real issue! the screenshot really made the difference, I understand perfectly what is going on.

>Actually, now that I am having a look at it again. I am not sure if the fab should show over the divider even when sticky

yes, the fab should show over a sticky divider as well

**Comment by manucorporat** [comment]
can you share the markup?

**Comment by alan-agius4** [comment]
z-index of the fab is `100`, while the one of the divider is `1000`

Markup below

```html
	<ion-fab>
		<button ion-fab>Button</button>
	</ion-fab>

	<ion-item-group>
		<ion-item-divider>Games</ion-item-divider>

		<ion-item>Pokémon Yellow</ion-item>
		<ion-item>Super Metroid</ion-item>
		<ion-item>Mega Man X</ion-item>
	</ion-item-group>
```

**Comment by manucorporat** [comment]
`<ion-item-group>` should be inside `<ion-list>`. is that right @brandyscarney ?

**Comment by alan-agius4** [comment]
Not as per documention
http://ionicframework.com/docs/v2/components/#inset-list

**Comment by manucorporat** [comment]
@alan-agius4 so here's the thing. I understand the issue, but I can't reproduce it!! it works ok in my machine surprisingly.

<img width="367" alt="screen shot 2016-10-05 at 18 20 49" src="https://cloud.githubusercontent.com/assets/127379/19121901/7a82490a-8b28-11e6-85fe-e727b084d575.png">

**Comment by alan-agius4** [comment]
Thats odd, you are using RC 0? And same markup?

**Comment by manucorporat** [comment]
```
<ion-header>
  <ion-toolbar>
    <ion-title>Floating Action Buttons</ion-title>
  </ion-toolbar>
</ion-header>

<ion-content>
    <ion-fab>
        <button ion-fab>Button</button>
    </ion-fab>

    <ion-item-group>
        <ion-item-divider sticky>Games</ion-item-divider>

        <ion-item>Pokémon Yellow</ion-item>
        <ion-item>Super Metroid</ion-item>
        <ion-item>Mega Man X</ion-item>
    </ion-item-group>
</ion-content>
```

![screen shot 2016-10-05 at 18 36 47](https://cloud.githubusercontent.com/assets/127379/19122434/c63b4a5c-8b2a-11e6-8c15-0aaaa4cd9e25.png)

**Comment by alan-agius4** [comment]
Seems like you managed to replicate it now :)

**Comment by manucorporat** [comment]
@alan-agius4 I don't think so... the fab button is over the divider

**Comment by alan-agius4** [comment]
Right, well, i dont know :/ why i am having that issue than.

**Comment by manucorporat** [comment]
@alan-agius4 We are both missing something for sure haha!

- can you create a repo that reproduces the issue?
- are you using tabs?
- is the fab ALWAYS under the divider?
- are there some kind of artifacts when clicking the fab, or when the page is transitioning?
- any custom css applied to fab/content/list/divider?

**Comment by alan-agius4** [comment]
Found the issue :), I have the ion-fab button inside a custom component.
For this case I wrapped it inside a "div" to mimic the issue.

Repo:
https://github.com/alan-agius4/ionic-playground

**Comment by alan-agius4** [comment]
Found the issue :), I have the ion-fab button inside a custom component.
For this case I wrapped it inside a "div" to mimic the issue.

Repo:
https://github.com/alan-agius4/ionic-playground

File:
https://github.com/alan-agius4/ionic-playground/blob/master/src/pages/hello-ionic/hello-ionic.html

Also, even something like the below will cause the issue

` <template [ngIf]="1 === 1">
    <ion-fab>
      <button ion-fab>Button</button>
    </ion-fab>
  </template>
`

**Comment by manucorporat** [comment]
fixed! 14e668c 

I didn't merge your PR since it didn't fix the root of the problem, but thank you anyway, your help resolving this issue has been amazing!

**Comment by alan-agius4** [comment]
cool :)
